src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -729,6 +727,20 @@ CHAMELEON.CORE.MTTableEditor.initTabs = function () {
     });
 };
 
+CHAMELEON.CORE.MTTableEditor.mapChameleonMessageTypeToBootstrapStyle = function (chameleonMessageTypeName) {
+    chameleonMessageTypeName = chameleonMessageTypeName.toLowerCase();
+
+    if ('error' === chameleonMessageTypeName) {
+        return 'danger'
```

Comment:
Please add a semicolon (also in line 738).

Reply:
Whoops. Done.